ouroboros-consensus/docs/interface-CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ https://keepachangelog.com/en/1.1.0/, adapted to our plan explained above.
 
 - New supported node to client version `NodeToClientV_14` with new queries:
   - `GetPoolState`: Get the pool state for the given stake pool ids
+  - `GetStakeSnapshot`: Get the snapshot of stake distribution
```

Comment:
Why do we write this entry here? Why not in a section above `2022-07-20`?

Reply:
I've moved it to a new section.